README.md

```diff
@@ -125,6 +125,8 @@ upstream proxy server. Zero or negative value disables it. Default is 0 (disable
 `Transport` can be either `direct` or `CDN`. If the server host wishes you to connect to it directly, use `direct`. If
 instead a CDN is used, use `CDN`.
 
+`RemoteHost` required if direct connection is used. Format: `<IP or domain name>[:port]`
```

Comment:
`RemoteHost` is configurable from the config file (so it should be documented), but it isn't required to be in the config file, as it could be supplied through the environment variable (used by Shadowsocks) or through the CLI options. The config file parser is called after these sources for `RemoteHost` are checked

https://github.com/cbeuw/Cloak/blob/master/cmd/ck-client/ck-client.go#L142